controller/app/cmdline/src/cmd_line.cpp

```diff
@@ -1015,10 +1043,11 @@ int cmd_line::cmd_view_media_clock(int total_matched, std::vector<cli_argument*>
                 if (stream_output_desc)
                 {
                     avdecc_lib::stream_output_descriptor_response *stream_output_resp_ref = stream_output_desc->get_stream_output_response();
+                    avdecc_lib::descriptor_response_base *desc_base_resp = stream_output_desc->get_descriptor_response();
                     is_clock_sync_source_set = stream_output_resp_ref->stream_flags_clock_sync_source();
                     if(is_clock_sync_source_set)
                     {
-                        desc_obj_name = stream_output_resp_ref->object_name();
+                        desc_obj_name = desc_base_resp->object_name();
```

Comment:
Yes, this is unnecessary. Will make the changes to have the desc responses inherit rather than forward declare desc response base this afternoon.



> On Mar 18, 2015, at 1:01 PM, andrew-elder <notifications@github.com> wrote:
> 
> In controller/app/cmdline/src/cmd_line.cpp:
> 
> >                      is_clock_sync_source_set = stream_output_resp_ref->stream_flags_clock_sync_source();
> >                      if(is_clock_sync_source_set)
> >                      {
> > -                        desc_obj_name = stream_output_resp_ref->object_name();
> > +                        desc_obj_name = desc_base_resp->object_name();
> Don't think this edit should be required?
> Doesn't stream_outupt_descriptoer_response derive from descriptor_response_base, so should be able to call ->object_name on the derived class?
> 
> —
> Reply to this email directly or view it on GitHub.
>